travis/jekyll_build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Please also add the License to any new files that aren't documentation. Fix throughout. Thanks.

Reply:
I'm sorry, I'm not understanding this: specifically, what license and where? Only thing I can think of is `_plugins/` but since that's all custom code built specifically for us, it should share the license of the repository.

Reply:
Apologies on being vague. Where we can we'll need to add our Apache 2.0 License.

https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/setup.py#L1

Reply:
You'll also need to chmod 755 all of these shell scripts otherwise Travis won't be able to run them.